webpack.config.js

```diff
@@ -97,7 +97,7 @@ jsLoader = {
 };
 
 if ( CALYPSO_ENV === 'development' ) {
-	webpackConfig.plugins.push( new PragmaCheckPlugin() );
+//	webpackConfig.plugins.push( new PragmaCheckPlugin() );
```

Comment:
Problem: `state/themes/actions.js` is replaced with noop in `webpack.config.node.js` but the pragma checker doesn't know this, so we still get the `not /** @ssr-ready **/` error. @ehg: any idea how we should handle this?